docs/time_series_causal_graph.md

```diff
@@ -0,0 +1,420 @@
+from cai_causal_graph import TimeSeriesCausalGraph
+
+ts_cg: TimeSeriesCausalGraph
+
+# Via identifier
+ts_cg.replace_node(node_id='X lag(n=3)', new_node_id='Y lag(n=3)')
+# Via variable name and time lag
+ts_cg.replace_node(node_id='X lag(n=3)', time_lag=3, variable_name='Y')
+
+# Variable type can also be specified
+ts_cg.replace_node(node_id='X lag(n=3)', new_node_id='Y lag(n=3)', variable_type=NodeVariableType.UNSPECIFIED)
+```
+
+## Delete nodes and edges
+You can delete nodes and edges from the time series graph.
+
+```python
+from cai_causal_graph import EdgeType, TimeSeriesCausalGraph
+
+ts_cg: TimeSeriesCausalGraph
+
+# Delete node
+ts_cg.delete_node(identifier='X lag(n=3)')
+
+# Delete edge (the edge type can be specified if desired)
+ts_cg.delete_edge(source='X lag(n=3)', destination='Z lag(n=3)', edge_type=EdgeType.DIRECTED_EDGE)
+
+# Delete edge from pair (the edge type can be specified if desired)
+ts_cg.remove_edge_by_pair(pair=('X lag(n=3)', 'Z lag(n=3)'), edge_type=EdgeType.DIRECTED_EDGE)
+```
+
+## Main properties
+### Adjacency_matrices
+Return the adjacency matrix dictionary of the minimal causal graph.
+The keys are the time deltas and the values are the adjacency matrices.
+
+```python
+from cai_causal_graph import TimeSeriesCausalGraph
+
+ts_cg: TimeSeriesCausalGraph
+
+adjacency_matrices = ts_cg.adjacency_matrices
+```
+
+### maxlag
+Return the autoregressive order of the time series graph, i.e. the maximum lag of the nodes in the minimal graph.
+
+```python
+from cai_causal_graph import TimeSeriesCausalGraph
+
+ts_cg: TimeSeriesCausalGraph
+
+maxlag = ts_cg.maxlag
+```
+
+### Variables
+Return the list of variable identifiers in the time series graph.
+Variables differ from nodes in that they do not contain the lag.
+For example, if the graph contains the node "X1 lag(n=2)", the variable is "X1".
+
+```python
+from cai_causal_graph import TimeSeriesCausalGraph
+
+ts_cg: TimeSeriesCausalGraph
+
+variables = ts_cg.variables
+```
+### Stationarity
+
+It may be useful to check whether the time series graph is stationary, i.e. its edges are not
+dependent on the corresponding time lags. In other words, the graph is the same for all time lags.
+Stationarity is a useful concept for future prediction: if the graph is stationary, it can be used to predict the future
+for any time lag.
+Conversely, if the graph is not stationary, it can only be used to predict the future for the specified time lags.
+Thus, in a stationary graph, if the edge `X lag(n=1) -> Y lag(n=1)` exists, also the edge `X lag(n=2) -> Y lag(n=2)` 
+must be present if the nodes `X lag(n=2)` and `Y lag(n=2)` are also present.
+
+You can check whether the graph is stationary by calling the `is_stationary_gragh` method.
+
+```python
+
+from cai_causal_graph import TimeSeriesCausalGraph
+
+ts_cg: TimeSeriesCausalGraph
+
+is_stationary = ts_cg.is_stationary_graph()
+```
+
+Moreover, you can make it stationary by calling the `get_stationary_graph` method as follows:
```

Comment:
@matteo-causalens Could you add a line explaining how info is propagated from `t=0`?